### PR TITLE
feat!: v37

### PR DESCRIPTION
**Pull request by rarkins**
## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by rarkins — COMMENTED** [review]
Not ready for merge

**Review by viceice — CHANGES_REQUESTED** [review]
😉

File: docs/usage/self-hosted-configuration.md [hunk 1]
```diff
@@ -830,7 +830,7 @@ Secret names must start with an upper or lower case character and can have only
 
 By default, Renovate will use the most efficient approach to updating package files and lock files, which in most cases skips the need to perform a full module install by the bot.
 If this is set to false, then a full install of modules will be done.
-This is currently applicable to `npm` and `lerna`/`npm` only, and only used in cases where bugs in `npm` result in incorrect lock files being updated.
+This is currently applicable to `npm` only, and only used in cases where bugs in `npm` result in incorrect lock files being updated.
```
Comment (HonkingGoose) [inline docs/usage/self-hosted-configuration.md:833]: Are you sure the PR title is correct? I think we're missing an `!` somewhere, to denote the `major` release that the Semantic Release bot should create after this PR gets merged into `main`?

Comment (rarkins) [reply docs/usage/self-hosted-configuration.md:833]: So far this v37 branch contains only one commit/change, but it will be expanded on

**Pull request closed without merging**